lib/api/src/modules/stories.ts

```diff
@@ -382,6 +385,38 @@ export const init: ModuleFn = ({
       (storiesHash[id] as Story).args = args;
       store.setState({ storiesHash });
     });
+
+    fullAPI.on(CURRENT_STORY_WAS_SET, function handleSetStory({ storyId, viewMode }) {
```

Comment:
We already have events to indicate that configuring is done:
  A) In 6.0, the `SET_STORIES` event, which is guaranteed to run once when the configure process has run. I'm ok with adding `error` to it though if that makes sense.

  B) In 5.x the `STORIES_CONFIGURED` event, I'm not sure what runs when there is an error however.